optuna_dashboard/ts/components/Artifact/TableArtifactViewer.tsx

```diff
@@ -115,6 +121,16 @@ export const useTableArtifactModal = (): [
   return [openModal, renderDeleteStudyDialog]
 }
 
+const loadData = (props: TableArtifactViewerProps): Promise<Data[]> => {
+  if (props.filetype === "csv") {
+    return loadCSV(props)
+  } else if (props.filetype === "jsonl") {
+    return loadJsonl(props)
```

Comment:
Seems that for the json file:
```jsonl
{"a": 3}
{"b": 3}
```
the table is not displaying correctly (the line `{"b": 3}` is ignored). How about filling in missing values with `-` or something, and making the columns a union of all attributes of jsonl?

Reply:
Just a comment: It seems that for a jsonl file like `{"a": {"b": 4, "c": 5}, "v": 6}`, it is not flattened properly. However, addressing this might be too complex and beyond the scope of this PR, so I think it's ok to ignore it for now.

Reply:
I followed the 1st comment. But, for the 2nd comment, I talked it with @c-bata, and we will do it for another PR.